packages/flutter/lib/src/foundation/stack_frame.dart

```diff
@@ -0,0 +1,152 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show hashValues;
+
+import 'package:meta/meta.dart';
+
+/// A object representation of a frame from a stack trace.
+///
+/// {@tool sample}
```

Comment:
@gspencergoog

Reply:
This is correct (for now).  I'll swap it to be "snippet" in my PR after you land.

Reply:
...or you will after I land, depending on who gets there first!  It will cause an analysis error if you forget, so it shouldn't be too hard to find.